src/PolyhedralGeometry/PolyhedralComplex/properties.jl

```diff
@@ -609,7 +609,7 @@ lineality_dim(PC::PolyhedralComplex) = pm_object(PC).LINEALITY_DIM::Int
 @doc raw"""
     f_vector(PC::PolyhedralComplex)
 
-Compute the vector $(f₀,f₁,f₂,...,f_{dim(PC))$` where $f_i$ is the number of
+Compute the vector $(f₀,f₁,f₂,...,f_{dim(PC)})$ where $f_i$ is the number of
```

Comment:
How does this look in a PDF? I hope it looks awful.

Reply:
Yes. It actually skips the "unicode indices".
![Bildschirmfoto vom 2024-08-11 14-47-52](https://github.com/user-attachments/assets/b51e3e31-7559-432a-9e4d-2ade22a33f26)